src/nucname.cpp

```diff
@@ -6,8 +6,148 @@
 #ifndef PYNE_IS_AMALGAMATED
 #include "nucname.h"
 #include "state_map.cpp"
+#include <iostream>
 #endif
 
+/*** Constructs the LL to element name Dictionary ***/
+pyne::nucname::name_elt_t pyne::nucname::get_name_elt() {
+  pyne::nucname::name_elt_t lname;
+
+  lname["H"] = "Hydrogen";
+  lname["He"] = "Helium";
+  lname["Li"] = "Lithium";
+  lname["Be"] = "Beryllium";
+  lname["B"] = "Boron";
+  lname["C"] = "Carbon";
+  lname["N"] = "Nitrogen";
+  lname["O"] = "Oxygen";
+  lname["F"] = "Fluorine";
+  lname["Ne"] = "Neon";
+  lname["Na"] = "Sodium";
+  lname["Mg"] = "Magnesium";
+  lname["Al"] = "Aluminium";
```

Comment:
What is the basis for these spellings.  Are they based on something from Geant?  They may not be universal....